samples/python/agents/pydantic_agent/agent.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel, Field
```

Comment:
`pydantic` and `pydantic_ai` both need to be added as dependencies? [In pyproject.toml](https://github.com/google/A2A/blob/cd9eb5e66219e4afab4e35985b2514bda5943b1f/samples/python/pyproject.toml)